translations/OpenOrienteering_sv.ts

```diff
@@ -438,12 +438,12 @@
     <message>
         <location filename="../src/color_dock_widget.cpp" line="48"/>
         <source>CMYK</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
It's by mistake, updating it right away...
On Sep 21, 2015 7:44 AM, "Kai Pastor" <notifications@github.com> wrote:

> In translations/OpenOrienteering_sv.ts
> <https://github.com/OpenOrienteering/mapper/pull/518#discussion_r39942485>
> :
>
> > @@ -438,12 +438,12 @@
> >      <message>
> >          <location filename="../src/color_dock_widget.cpp" line="48"/>
> >          <source>CMYK</source>
> > -        <translation type="unfinished"></translation>
> > +        <translation></translation>
>
> I need to test if empty entries result in empty or untranslated ouput.
> However, it is at least difficult to tell whether the missing entry is
> intentionally or by mistake.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenOrienteering/mapper/pull/518/files#r39942485>.
>